Cargo.toml

```diff
@@ -1,17 +1,20 @@
 [package]
 name = "trin"
 version = "0.1.0"
-authors = ["Jacob Kaufmann <jacobkaufmann18@gmail.com>", "Jason Carver <ut96caarrs@snkmail.com>"]
+authors = ["https://github.com/ethereum/trin/graphs/contributors"]
```

Comment:
Not trying to step on any toes w/ this change, happy to revert to individual contributors, but this seems like a more sustainable approach